actors/builtin/miner/monies.go

```diff
@@ -169,3 +173,7 @@ func ConsensusFaultPenalty(thisEpochReward abi.TokenAmount) abi.TokenAmount {
 		big.NewInt(builtin.ExpectedLeadersPerEpoch),
 	)
 }
+
+func LockedRewardFromRewardV6(reward abi.TokenAmount) abi.TokenAmount {
+	return big.Div(big.Mul(reward, LockedRewardFactorNumV6), LockedRewardFactorDenomV6)
```

Comment:
My preferred pattern here is to pass the network version as a parameter here rather than switch on it at the caller. In general, this would reduce the number of places that need to get the version comparison right, and statically find all the relevant places. (because this is introducing a new function, previously identity, we don't get the static win here).

Reply:
Done.